packages/client/hmi-client/src/services/model.ts

```diff
@@ -10,17 +10,25 @@ import { fileToJson } from '@/utils/file';
 import { Ref } from 'vue';
 import { DateOptions } from './charts';
 
-export async function createModel(model: Model): Promise<Model | null> {
+export async function createModel(
+	model: Model,
+	modelConfigurationId?: ModelConfiguration['id']
+): Promise<Model | null> {
 	delete model.id;
-	const response = await API.post(`/models`, model);
+	const response = await API.post(`/models`, { model, modelConfigurationId });
```

Comment:
It might make more coherent sense to send `modelConfigurationId` as a query parameter than a part of the request body. eg

```
POST /models?modelConfigurationId=xyzxyz
```

It is a little weird to mix, but I think it conveys a better idea of what the route is about, that is, posting a new model constrained by an optional model-configuration.